NFIQ2/NFIQ2Algorithm/src/NFIQ2Algorithm.cpp

```diff
@@ -35,6 +35,18 @@ NFIQ::NFIQ2Algorithm::computeQualityScore(
              qualityFeatureSpeed ) );
 }
 
+std::list<NFIQ::QualityFeatureData> NFIQ::NFIQ2Algorithm::computeQualityFeatures(
+  const NFIQ::FingerprintImageData& rawImage )
+{
+	std::list<NFIQ::ActionableQualityFeedback> actionableQuality;
+	std::list<NFIQ::QualityFeatureSpeed> qualityFeatureSpeed;
+  
+	return ( this->pimpl->computeQualityFeatures(
```

Comment:
`astyle` defect tagged by Travis CI:
```suggestion
  return ( this->pimpl->computeQualityFeatures(
```

Reply:
I wasn't aware that you already activated astyle. I was looking for your commit to check in the initial astyled files.